openshift/release/generate-release.sh

```diff
@@ -3,8 +3,14 @@
 source $(dirname $0)/resolve.sh
 
 release=$1
-
-quay_image_prefix="quay.io/openshift-knative/knative-serving-"
 output_file="openshift/release/knative-serving-${release}.yaml"
 
-resolve_resources config/ $output_file $quay_image_prefix $release
\ No newline at end of file
+if [ $release = "ci" ]; then
+    image_prefix="image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-"
+    tag=""
+else
+    image_prefix="quay.io/openshift-knative/knative-serving-"
+    tag=$release
+fi
```

Comment:
Should we make those passable to the script?

Reply:
I can do that, not a problem. I havent dobe it because the result would be basically the same as it currently is on the line 16.

Reply:
You're right, let's merge as-is and refactor if needed.